apps/prairielearn/src/lib/db-types.ts

```diff
@@ -1042,7 +1054,7 @@ export const SubmissionSchema = z.object({
   params: z.record(z.string(), z.any()).nullable(),
   partial_scores: z.record(z.string(), z.any()).nullable(),
   raw_submitted_answer: z.record(z.string(), z.any()).nullable(),
-  regradable: z.boolean().default(false),
+  regradable: z.boolean(),
```

Comment:
For correctness/consistency:

```suggestion
  regradable: z.boolean().nullable(),
```